File: Beginner/1073 - Even Square/1073.c
#include<stdio.h>

int main(){
	
	int max, i;
	
	scanf("%i", &max);
	
	for(i=2; i<=max; i++){
		if(i%2 == 0)
			printf("%i^2 = %i\n", i, i*i);
	}
	
	return 0;
}

File: Beginner/1073 - Even Square/1073.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int max = int.Parse(Console.ReadLine());

        for (int i = 2; i <= max; i += 2)
        {
            Console.WriteLine($"{i}^2 = {i * i}");
        }
    }
}